libmamba/include/mamba/download/request.hpp

```diff
@@ -140,6 +140,11 @@ namespace mamba::download
     {
         std::string mirror_name;
         std::string url_path;
+        // TODO maybe we would want to use a struct instead
+        // containing the `checksum` and its `type`
+        // (to handle other checksums types like md5...)
+        // cf. `Checksum` struct in powerloader
+        std::string sha256;
```

Comment:
I think we want something even more generic to handle additional parameters specific to a kind of storage. This should be done in a dedicated PR, though.